.circleci/create_circleci_config.py

```diff
@@ -138,11 +117,10 @@ def to_dict(self):
         )
 
         steps.append({"run": {"name": "Create `test-results` directory", "command": "mkdir test-results"}})
-
         test_command = ""
         if self.command_timeout:
             test_command = f"timeout {self.command_timeout} "
-        test_command += f"python -m pytest -rs --junitxml=test-results/junit.xml -n {self.pytest_num_workers} " + " ".join(pytest_flags)
+        test_command += f"python3 -m pytest -rsfE -p no:warnings -o junit_family=xunit1 --tb=short --junitxml=test-results/junit.xml -n {self.pytest_num_workers} " + " ".join(pytest_flags)
```

Comment:
maybe just a bit of explanation of `junit_family` (and others if you want) to people could understand easily without need to check the doc (if not really necessary)

Reply:
Sure! junit is to prepare for the split that will come next. Junit 2 is incompatible